sec.club/src/components/EventBanner/EventBanner.scss

```diff
@@ -0,0 +1,70 @@
+@import "../../variables.scss";
+
+
+
+.flyer{
+    padding-bottom: 3vh;
+
+    @media (max-width: $sm-max) {
+        text-align: center;
+        align-content: center;
+    }
+
+    .container {
+        display: flex;  
+        border: $primary-color 1px solid;
+
+        @media (max-width: $sm-max) {
+            margin: auto;
+        }
+
+        img {
+            max-width: 50%;
+            max-height: 60vh;
```

Comment:
Don't use viewport height units for something like this, it's harder to work with than you'd think. Stick with `px`

Reply:
Actually `%` might be better?